include/oxen/quic/endpoint.hpp

```diff
@@ -230,9 +207,11 @@ namespace oxen::quic
         std::vector<ustring> inbound_alpns;
         std::chrono::nanoseconds handshake_timeout{DEFAULT_HANDSHAKE_TIMEOUT};
 
-        std::map<ustring, ustring> anti_replay_db;
-        std::map<ustring, ustring> encoded_transport_params;
-        std::map<ustring, ustring> path_validation_tokens;
+        std::unordered_map<ustring_view, gtls_ticket_ptr> session_tickets;
```

Comment:
The key here being a string_view seems dangerous: who owns the underlying string the view is pointing into and how do we guarantee it stays valid as long as the key exists in `session_tickets`?

Reply:
Additionally: a string view key is also dangerous as a key because if the underlying string *changes* the unordered_map gets broken.

Reply:
To clarify: if this *is* safe (which I suspect it is) this deserves a comment explaining why it's safe.

Reply:
I think this is perfectly fine now with my added commit: I left a comment specifically about the safety of using the key as a map.

Reply:
Yes thank you. This was intended safe as enforced usage of `gtls_ticket_ptr` in a unique ptr using the constructor, but your commit more properly parametrized that